### DISCUSSION TITLE
科技爱好者周刊（第 298 期）：轮到硬件工程师吃香了 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#39] [General]
# 科技爱好者周刊（第 298 期）：轮到硬件工程师吃香了 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-298

**Comment by luhuadong** [comment]
看完之后只记住了要穿五趾袜，哈哈哈

**Reply by penggeon** [reply]
没想到还有人会记得这个网站（小声碎碎念
